Sming/Libraries/SDCard/README.md

```diff
@@ -1,3 +1,27 @@
 # SD Card
 
 Low-level support code for accessing SD Cards using FATFS.
+
+I couldn't get SDCard to work on a Wemos D1 mini with an SD Card adaptor. Always gave the same error. SD Card Error: FF.
```

Comment:
Please, remove this text from here. Here we should have helpful information about the SDCard library itself. Explanation of issues with the library is good as part of a PR ticket information but does not belong here.

Reply:
Please, leave the images as part of the ticket and remove them from this commit. Remove `samples/SDCard/SPI_MSBFIRST.JPG`  and `samples/SDCard/SPI_LSBFIRST.JPG`.

Reply:
text and images removed